src/pages/Admin/Charity/DonationsTable/DonationsTable.tsx

```diff
@@ -15,9 +15,9 @@ export default function DonationsTable({ classes = "" }) {
       <QueryLoader
         queryState={{ data: data?.Items, ...rest }}
         messages={{
-          loading: "Fetching donations..",
-          error: "Failed to get donations",
-          empty: "No donations found",
+          loading: "Fetching contributions..",
```

Comment:
@abbasali1994 We should use PAYMENT_WORDS here in this component too. Otherwise the Charity app will show 'contribution(s)' and not 'donation(s)'